### PR TITLE
feat(collect-modal): apply collect data modal

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- create and apply collect data modal
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+By default, the components use the information of the selected item when the modal is activated.
+Therefore, in order to use the components, the relevant information should be stored in the store.
+
+```ts
+import {
+    useCollectorDataModalStore,
+} from '@/services/asset-inventory/collector/shared/collector-data-modal/collector-data-modal-store';
+
+const collectorDataModalStore = useCollectorDataModalStore();
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/README.md:13]: praise: wow!!!!!!!!!!!!!

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+```
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/README.md:14]: Praise: Really Kind!

**Review by sulmoJ — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue [hunk 3]
```diff
@@ -88,19 +74,15 @@ const state = reactive({
...
-    emit('refresh-collector-list');
+    collectorPageStore.$patch({
+        visibleCollectorModal: true,
+    });
+    collectorDataModalStore.$patch({
+        selectedCollector: {
+            ...props.item,
+            collector_id: props.item.collectorId,
+        },
+    });
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContentItem.vue:85]: todo: You have to use functional 😎patch😎 when its `object` type!

**Review by yuda110 — APPROVED** [review]
LGTM!!! I left a comment!!!

**Pull request merged**